scripts/test.log

```diff
@@ -0,0 +1,10 @@
+"Download section test"
```

Comment:
maybe more infos are useful for a log file, like: time, command run or something like that.

Reply:
This is where the output of the commands in the .yp file go. I could add some time format for it, but that will be later once I get the program working for continuous integration and scripts transferred to the new format.